src/data/partner_parties.h

```diff
@@ -1,26 +1 @@
-static const struct TrainerMon sParty_StevenPartner[] = {
-    {
-        .species = SPECIES_METANG,
-        .lvl = 42,
-        .nature = NATURE_BRAVE,
-        .iv = TRAINER_PARTY_IVS(31, 31, 31, 31, 31, 31),
-        .ev = TRAINER_PARTY_EVS(0, 252, 252, 0, 6, 0),
-        .moves = {MOVE_LIGHT_SCREEN, MOVE_PSYCHIC, MOVE_REFLECT, MOVE_METAL_CLAW},
-    },
-    {
-        .species = SPECIES_SKARMORY,
-        .lvl = 43,
-        .nature = NATURE_IMPISH,
-        .iv = TRAINER_PARTY_IVS(31, 31, 31, 31, 31, 31),
-        .ev = TRAINER_PARTY_EVS(252, 0, 0, 0, 6, 252),
-        .moves = {MOVE_TOXIC, MOVE_AERIAL_ACE, MOVE_PROTECT, MOVE_STEEL_WING},
-    },
-    {
-        .species = SPECIES_AGGRON,
-        .lvl = 44,
-        .nature = NATURE_ADAMANT,
-        .iv = TRAINER_PARTY_IVS(31, 31, 31, 31, 31, 31),
-        .ev = TRAINER_PARTY_EVS(0, 252, 0, 0, 252, 6),
-        .moves = {MOVE_THUNDER, MOVE_PROTECT, MOVE_SOLAR_BEAM, MOVE_DRAGON_CLAW},
-    }
-};
+
```

Comment:
any reason not to delete the file?

Reply:
Isn't trainer_parties.h also still in the repo? We could also delete both, it's as you prefer

Reply:
As-written it would need to stay. Maybe with a `#if !COMPETITIVE_SYNTAX` around the `#include` of that file it would work, not 100% sure. I think it would have been either worries about migrations, or worries about configs that made me keep that empty `trainer_parties.h`... but I'm not sure why that would actually be necessary now :thinking: